### PR TITLE
support metadata and Append Object

### PR DESCRIPTION
**Pull request by owarai**

### CODE REVIEW

File: src/main/java/com/qingstor/sdk/config/EvnContext.java [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
-// |
-// | Unless required by applicable law or agreed to in writing, software
-// | distributed under the License is distributed on an "AS IS" BASIS,
-// | WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// | See the License for the specific language governing permissions and
-// | limitations under the License.
-// +-------------------------------------------------------------------------
-
+/*
+ * Copyright (C) 2016 Yunify, Inc.
```
Comment (Xuanwo) [inline src/main/java/com/qingstor/sdk/config/EvnContext.java]: 2020 for now

File: src/main/java/com/qingstor/sdk/constants/QSConstant.java [hunk 2]
```diff
@@ -79,16 +79,18 @@
...
 
-
     public static int HTTPCLIENT_CONNECTION_TIME_OUT = 60; // Seconds
     public static int HTTPCLIENT_READ_TIME_OUT = 100; // Seconds
     public static int HTTPCLIENT_WRITE_TIME_OUT = 100; // Seconds
 
-    public static final String VIRTUAL_HOST_STYLE  = "virtual_host_style"; //default url style, like this: https://bucket-name.zone-id.qingstor.com/object-name
-    public static final String PATH_STYLE = "path_style"; //https://zone-id.qingstor.com/bucket-name/object-name
+    public static final String VIRTUAL_HOST_STYLE =
+            "virtual_host_style"; // default url style, like this:
```
Comment (Xuanwo) [inline src/main/java/com/qingstor/sdk/constants/QSConstant.java]: code style format?

File: src/main/java/com/qingstor/sdk/config/EvnContext.java (hunk 1 above)
Comment (owarai) [reply src/main/java/com/qingstor/sdk/config/EvnContext.java]: replace 2016 with 2020. Program files add header license automatically.

File: src/main/java/com/qingstor/sdk/constants/QSConstant.java (hunk 2 above)
Comment (owarai) [reply src/main/java/com/qingstor/sdk/constants/QSConstant.java]: Yes, we use spotless format all .java file in source dir. Format style use google-java-format's AOSP style(use 4 indents).

**Pull request merged**